relm4/src/factory/mod.rs

```diff
@@ -41,11 +41,14 @@ where
     uid_counter: u16,
 }
 
+#[allow(dead_code)] // As of 1.60 dead code analysis erroneously reports the Debug impl as dead
+#[derive(Debug)]
```

Comment:
Can we investigate further why this is marked as dead code?

Reply:
Absolutely, was planning to create an upstream bug report but didn't want to block this PR on that.